tests/test_status_change.py

```diff
@@ -1500,3 +1500,167 @@ async def test_automerge_label_triggered_by_added_to_pr():
     assert gh.patch_data == {
         "body": f"{data['pull_request']['body']}\n\nAutomerge-Triggered-By: @Mariatta"
     }
+
+
+async def test_automerge_label_removed_by_core_dev():
+    sha = "f2393593c99dd2d3ab8bfab6fcc5ddee540518a9"
+    data = {
+        "action": "unlabeled",
+        "pull_request": {
+            "user": {"login": "miss-islington"},
+            "labels": [
+                {"name": "awaiting merge"},
+                {"name": "CLA signed"},
+            ],
+            "head": {"sha": sha},
+            "number": 5547,
+            "title": "bpo-32720: Fixed the replacement field grammar documentation.",
+            "body": "\n\n`arg_name` and `element_index` are defined as `digit`+ instead of `integer`.\n\nAutomerge-Triggered-By: @miss-islington",
```

Comment:
Can you add test for case where for some reason the "Automerge-Triggered-By: @miss-islington" was not in the PR body? (for example if someone removed it manually).

Reply:
Good idea! Done in https://github.com/python/miss-islington/pull/360/commits/7a0f0b3eaeafc1f1f9f84e3a50f0cbb0af4186f7.